integrations/aws/aws-rds-monitoring/README.md

```diff
@@ -0,0 +1,106 @@
+## Overview
+
+<!-- Sensu Integration description; supports markdown -->
+
+The [AWS RDS] integration collects RDS metrics from [AWS CloudWatch], and alerts on various Lamda conditions.
+
+<!-- Provide a high level overview of the integration contents (e.g. checks, filters, mutators, handlers, assets, etc) -->
+
+This integration includes the following resources:
+
+* `aws-rds-metrics` [check]
+* `sensu/sensu-cloudwatch-check:0.3.0` [asset] 
+
+
+## Dashboards
+
+<!-- List of compatible dashboards w/ screenshots (supports png, jpeg, and gif images; relative paths only; e.g. `![](img/dashboard-1.png)` )-->
+
+<!-- This integration is compatible with the [{{dashboard_name}}][{{dashboard_link}}] (included w/ [Sensu Plus][sensu-plus]). -->
+
+<!-- ![](img/dashboard.png) -->
+
+There are no compatible dashboards for this integration.
+
+## Setup
+
+<!-- Sensu Integration setup instructions, including Sensu agent configuration and external component configuration -->
+<!-- EXAMPLE: what configuration (if any) is required in a third-party service to enable monitoring? -->
+1. This integration requires access to AWS CloudWatch APIs.
+
+   All forms of AWS authentication supported by the [AWS CLI] are supported, including [EC2 IAM Instance Profiles]. This integration requires read-only access to AWS CloudWatch (e.g. `arn:aws:iam::aws:policy/CloudWatchReadOnlyAccess`).
+
+   `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` secret(s) or environment variable(s) are not needed if the check(s) from this integration are run on a `sensu-agent` installed on an EC2 instance with an IAM Instance Profile containing the `arn:aws:iam::aws:policy/CloudWatchReadOnlyAccess`.
+
+[AWS CLI]: https://aws.amazon.com/cli/
+[EC2 IAM Instance Profiles]: https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_use_switch-role-ec2_instance-profiles.html
+
+## Plugins
+
+<!-- Links to any Sensu Integration dependencies (i.e. Sensu Plugins) -->
+- [sensu/sensu-cloudwatch-check][sensu-cloudwatch-check-bonsai] ([GitHub][sensu-cloudwatch-check-github])
+
+## Metrics & Events
+
+This integration collects several [Amazon CloudWatch metrics for Amazon RDS] (i.e. `AWS/RDS` metrics).
+Please refer to the [Amazon CloudWatch metrics for Amazon RDS] reference documentation for descriptions of each metric.
+
+[Overview of monitoring metrics in Amazon RDS] https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/MonitoringOverview.html
+[Amazon CloudWatch metrics for Amazon RDS]: https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/rds-metrics.html
```

Comment:
```suggestion
This integration collects several [Amazon CloudWatch metrics for Amazon RDS] (i.e. `AWS/RDS` metrics).
Please refer to the [Amazon CloudWatch metrics for Amazon RDS] reference documentation for descriptions of each metric.

[Overview of monitoring metrics in Amazon RDS]: https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/MonitoringOverview.html
[Amazon CloudWatch metrics for Amazon RDS]: https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/rds-metrics.html
```

Looks like the overview link isn't actually used. Did you mean to have both mentions in the paragraph link to the same page?